deb/openmediavault/workbench/src/styles.scss

```diff
@@ -99,6 +99,20 @@ $omv-dark-theme: mat.define-dark-theme(
   width: auto;
 }
 
+.mat-card.omv-w-xs {
```

Comment:
IMO this could be done more nicer with CSS `@media` rule.

Reply:
i think, the width is not depending on the screen-width at all, so @media is not really appropriate.
The width of the form should be dependent by it's contents.